packages/ui-components/src/components/toggle-button/toggle-button.types.ts

```diff
@@ -2,12 +2,12 @@ import { EventEmitter } from '@stencil/core';
 import { IAnchor } from '../../types';
 import { EIconName, EOtherIconName } from '../icon/icon.types';
 
-export interface IRadioButton extends IAnchor {
+export interface IToggleButton extends IAnchor {
 	/** (required) The value to be emitted upon click events */
 	value: string;
-	/** (optional) The button's icon. Only valid for radio button icon */
+	/** (optional) The button's icon. Only valid for toggle button icon */
 	icon?: EIconName | EOtherIconName;
-	/** (optional) The button's label. Only valid for radio button text */
+	/** (optional) The button's label. Only valid for toggle button text */
 	label?: string;
 	/** (optional) Sets the button's styling to be disabled and disables click events */
 	disabled?: boolean;
```

Comment:
seems this prop `disabled` can be removed from here since isn't used to define this state on the component.

Reply:
When we are using the `toggle button` component by itself, this is the only way to disable it:
![image](https://user-images.githubusercontent.com/55213469/223699289-121b3257-9bf5-49a7-8c18-948e8e4fae1c.png)

Reply:
yeap seems legit